.prettier.config.js

```diff
@@ -1,35 +1,35 @@
-'use strict';
+"use strict";
 
 module.exports = {
-  arrowParens: 'always',
+  arrowParens: "always",
   bracketSpacing: true,
-  endOfLine: 'lf',
+  endOfLine: "lf",
```

Comment:
Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/openinf/openinf.github.io/pullRequest?prid=9929905)
```suggestion
  endOfLine: 'lf',
```